### ISSUE TITLE
Handling equal votes may need a reconsideration

### ISSUE THREAD

**Issue by FarazFe** [benadida/helios-server#175]
I came accross a possible issue with helios/models and I wanted to know your opinion

    def one_question_winner(cls, question, result, num_cast_votes):
            """
            determining the winner for one question
            """
            # sort the answers , keep track of the index
            counts = sorted(enumerate(result), key=lambda(x): x[1])
            counts.reverse()
    
            the_max = question['max'] or 1
            the_min = question['min'] or 0

            # if there's a max > 1, we assume that the top MAX win
            if the_max > 1:
                   return [c[0] for c in counts[:the_max]]

             # if max = 1, then depends on absolute or relative
             if question['result_type'] == 'absolute':
                 if counts[0][1] >=  (num_cast_votes/2 + 1):
                        return [counts[0][0]]
                 else:
                     return []
             else:
                   # assumes that anything non-absolute is relative
                   return [counts[0][0]]    

So basically when:
the_max = 3
result = [5,7,4,4,4,2,1]

this function would return [1,0,4]
Even though we have 3 candidates with equal votes,one of them (kinda randomly) is announced as winner.
I think this behaviour needs a reconsideration.

**Comment by benadida** [comment]
Yeah, agreed, this was a quick hack. Happy to take a PR :)
